.github/workflows/package-release.yml

```
@@ -0,0 +1,122 @@
+---
+name: CI Pipeline
```

```suggestion
name: build-workflow
```

and name file build-workflow.yml or name: package-release whatever you prefer more.

---

Renamed as `build-workflow.yml`